moodler/moodle.py

```diff
@@ -153,6 +154,25 @@ def submissions_statistics(course_id, is_verbose=False, download_folder=None):
             unreleased=unreleased_amount,
         )._asdict()
 
+    lti_stats = get_lti_stats_by_course(course_id)
```

Comment:
```suggestion
    lti_stats = get_lti_stats_by_course(course_id)
```

hmm...course_id is not globally unique and we have to use a more specific identifier.
It may not be the case for the Stagmont team, but it will be an issue if multiple Moodle instances will use the same LTI instance.
Please create a bug and prioritize it for LTI.

Reply:
This function has to be called as soon as Lti Intergation is enabled.

Reply:
Ticket for the bug has been created.